comanage_utils.py

```diff
@@ -69,6 +69,14 @@ def mkauthstr(user, passwd):
     return encodebytes(raw_authstr.encode()).decode().replace("\n", "")
 
 
+def get_ldap_authtok(ldap_authfile):
+    if ldap_authfile is not None:
+        ldap_authtok = open(ldap_authfile).readline().rstrip("\n")
+    else:
```

Comment:
```suggestion
def get_ldap_authtok(ldap_authfile):
    if ldap_authfile is not None:
        with open(ldap_authfile) as ldap_auth_fh:
             ldap_authtok = ldap_auth_fh.readline().strip()
    else:
```

I'm not sure why we'd restrict the strip to trailing newlines. I imagine any leading/trailing whitespace would be problematic.